src/dropdown/dropdown.component.ts

```diff
@@ -57,16 +57,16 @@ import { ElementService } from "./../utils/utils.module";
 			'bx--dropdown--disabled bx--list-box--disabled': disabled,
 			'bx--dropdown--invalid': invalid
 		}">
-		<div
+		<button
 			type="button"
 			#dropdownButton
 			class="bx--list-box__field"
 			[ngClass]="{'a': !menuIsClosed}"
 			[attr.aria-expanded]="!menuIsClosed"
 			[attr.aria-disabled]="disabled"
-			(click)="disabled ? $event.stopPropagation() : toggleMenu()"
+			(click)="toggleMenu()"
 			(blur)="onBlur()"
-			[attr.disabled]="disabled ? true : null">
+			[disabled]="disabled ? true : null">
```

Comment:
A bit weird this line but I guess there aren't many things you can do when dealing with the `disabled` attr